File: src/tasks/cli/command/remove/RemoveArguments.java
package tasks.cli.command.remove;

import omnia.data.structure.List;
import tasks.cli.command.common.simple.SimpleArguments;
import tasks.model.Task;

/** Model for parsed Remove command arguments. */
public final class RemoveArguments extends SimpleArguments {
  RemoveArguments(List<Task> tasks) {
    super(tasks);
  }

  /** The tasks to remove from the store. */
  @Override
  public List<Task> tasks() {
    return super.tasks();
  }
}

File: cli/src/me/strangepan/tasks/cli/feature/remove/RemoveArguments.kt
package me.strangepan.tasks.cli.feature.remove

import omnia.data.structure.List
import me.strangepan.tasks.cli.command.common.simple.SimpleArguments
import me.strangepan.tasks.engine.model.Task

/** Model for parsed Remove command arguments.  */
class RemoveArguments internal constructor(tasks: List<Task>, private val force: Boolean) : SimpleArguments(tasks) {
  /** Force. auto-confirm deletions, bypassing manual confirmations.  */
  fun force(): Boolean {
    return force
  }
}